src/components/world/worldHome.js

```diff
@@ -0,0 +1,26 @@
+import { React } from 'react';
+import PropTypes from 'prop-types';
+import AllCountries from '../countries/allCountries';
+import WorldData from './worldData';
+
+const WorldHome = (props) => {
```

Comment:
According to the project requirements:

![image](https://user-images.githubusercontent.com/57421823/157003157-9a9afcd2-9554-41e0-a08c-5d441b40eb6a.png)

Therefore:

- [ ] Please, kindly implement a filter for your main categories (in this case, those categories are the countries).  

## Hints

This filter could be a search box or a drop-down list.  The main goal is that only the categories that apply appear when filtered.  In your case, since the categories are the countries, and there are a lot, a search box would make more sense.